### ISSUE TITLE
Depreciations: ImageMagick 6 Depreciations With Node 10.6.0

### ISSUE THREAD

**Issue by 3DEsprit** [elad/node-imagemagick-native#185]
**Issue**: It seems there are a few depreciation warnings when rebuilding im-native 1.9.3 with Node 10.6.0. It looks like it's mostly ImageMagick 6 compatibility, and figured it would be something to mention since Node 11 may completely depreciate these changes.

On a side note, while i have not tried a rebuild with Node v9, there are no issues with Node v8.

```
  CXX(target) Release/obj.target/imagemagick/src/imagemagick.o
../src/imagemagick.cc:465:24: warning: 'Call' is deprecated [-Wdeprecated-declarations]
    context->callback->Call(2, argv);
                       ^
../../nan/nan.h:1646:3: note: 'Call' has been explicitly marked deprecated here
  NAN_DEPRECATED inline v8::Local<v8::Value>
  ^
../../nan/nan.h:102:40: note: expanded from macro 'NAN_DEPRECATED'
# define NAN_DEPRECATED __attribute__((deprecated))
                                       ^
../src/imagemagick.cc:558:10: warning: 'Utf8Value' is deprecated [-Wdeprecated-declarations]
        *String::Utf8Value(resizeStyleValue) : "aspectfill";
         ^
/Users/dbrown/.node-gyp/10.6.0/include/node/v8.h:2851:5: note: 'Utf8Value' has been explicitly marked deprecated here
    V8_DEPRECATED("Use Isolate version",
    ^
/Users/dbrown/.node-gyp/10.6.0/include/node/v8config.h:327:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/imagemagick.cc:562:10: warning: 'Utf8Value' is deprecated [-Wdeprecated-declarations]
        *String::Utf8Value(gravityValue) : "Center";
         ^
/Users/dbrown/.node-gyp/10.6.0/include/node/v8.h:2851:5: note: 'Utf8Value' has been explicitly marked deprecated here
    V8_DEPRECATED("Use Isolate version",
    ^
/Users/dbrown/.node-gyp/10.6.0/include/node/v8config.h:327:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/imagemagick.cc:566:4: warning: 'Utf8Value' is deprecated [-Wdeprecated-declarations]
                *String::Utf8Value(formatValue) : "";
                 ^
/Users/dbrown/.node-gyp/10.6.0/include/node/v8.h:2851:5: note: 'Utf8Value' has been explicitly marked deprecated here
    V8_DEPRECATED("Use Isolate version",
    ^
/Users/dbrown/.node-gyp/10.6.0/include/node/v8config.h:327:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/imagemagick.cc:570:10: warning: 'Utf8Value' is deprecated [-Wdeprecated-declarations]
        *String::Utf8Value(srcFormatValue) : "";
         ^
/Users/dbrown/.node-gyp/10.6.0/include/node/v8.h:2851:5: note: 'Utf8Value' has been explicitly marked deprecated here
    V8_DEPRECATED("Use Isolate version",
    ^
/Users/dbrown/.node-gyp/10.6.0/include/node/v8config.h:327:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/imagemagick.cc:574:10: warning: 'Utf8Value' is deprecated [-Wdeprecated-declarations]
        *String::Utf8Value(filterValue) : "";
         ^
/Users/dbrown/.node-gyp/10.6.0/include/node/v8.h:2851:5: note: 'Utf8Value' has been explicitly marked deprecated here
    V8_DEPRECATED("Use Isolate version",
    ^
/Users/dbrown/.node-gyp/10.6.0/include/node/v8config.h:327:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/imagemagick.cc:578:10: warning: 'Utf8Value' is deprecated [-Wdeprecated-declarations]
        *String::Utf8Value(backgroundValue) : "";
         ^
/Users/dbrown/.node-gyp/10.6.0/include/node/v8.h:2851:5: note: 'Utf8Value' has been explicitly marked deprecated here
    V8_DEPRECATED("Use Isolate version",
    ^
/Users/dbrown/.node-gyp/10.6.0/include/node/v8config.h:327:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/imagemagick.cc:669:24: warning: 'Call' is deprecated [-Wdeprecated-declarations]
    context->callback->Call(2, argv);
                       ^
../../nan/nan.h:1646:3: note: 'Call' has been explicitly marked deprecated here
  NAN_DEPRECATED inline v8::Local<v8::Value>
  ^
../../nan/nan.h:102:40: note: expanded from macro 'NAN_DEPRECATED'
# define NAN_DEPRECATED __attribute__((deprecated))
                                       ^
../src/imagemagick.cc:1035:11: warning: 'Utf8Value' is deprecated [-Wdeprecated-declarations]
         *String::Utf8Value(gravityValue) : "";
          ^
/Users/dbrown/.node-gyp/10.6.0/include/node/v8.h:2851:5: note: 'Utf8Value' has been explicitly marked deprecated here
    V8_DEPRECATED("Use Isolate version",
    ^
/Users/dbrown/.node-gyp/10.6.0/include/node/v8config.h:327:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
9 warnings generated.
  SOLINK_MODULE(target) Release/imagemagick.node
```

**Comment by cartuchogl** [comment]
Fixed by #186

**Comment by 3DEsprit** [comment]
You rock! Thank you so much!
